### PR TITLE
chore: check if checkbox is checked

### PR DESCRIPTION
**Pull request by debs-obrien**

### CODE REVIEW

File: assets/demo-todo-app.spec.js [hunk 1]
```diff
@@ -168,13 +168,20 @@ test.describe('Item', () => {
 
     const firstTodo = page.getByTestId('todo-item').nth(0);
     const secondTodo = page.getByTestId('todo-item').nth(1);
-    await firstTodo.getByRole('checkbox').check();
+    const firstTodoCheck = firstTodo.getByRole('checkbox');
+    const secondTodoCheck = secondTodo.getByRole('checkbox');
+
+    await firstTodoCheck.check();
+    await expect(firstTodoCheck).toBeChecked();
```
Comment (mxschmitt) [inline assets/demo-todo-app.spec.js:175]: I feel like this assertion does not make a lot of sense, isn't asserting for the completed class already enoughj?

Comment (debs-obrien) [reply assets/demo-todo-app.spec.js:175]: ok perhaps. no idea to be honest.

File: assets/demo-todo-app.spec.js [hunk 2]
```diff
@@ -168,12 +168,15 @@ test.describe('Item', () => {
 
     const firstTodo = page.getByTestId('todo-item').nth(0);
     const secondTodo = page.getByTestId('todo-item').nth(1);
-    await firstTodo.getByRole('checkbox').check();
+    const firstTodoCheckbox = firstTodo.getByRole('checkbox');
+    const secondTodoCheckbox = secondTodo.getByRole('checkbox');
```
Comment (dgozman) [inline assets/demo-todo-app.spec.js:172]: Unused?

File: assets/demo-todo-app.spec.js [hunk 3]
```diff
@@ -317,8 +320,10 @@ test.describe('Persistence', () => {
     }
 
     const todoItems = page.getByTestId('todo-item');
-    await todoItems.nth(0).getByRole('checkbox').check();
+    const firstTodoCheck = todoItems.nth(0).getByRole('checkbox')
```
Comment (dgozman) [inline assets/demo-todo-app.spec.js:323]:
```suggestion
    const firstTodoCheck = todoItems.nth(0).getByRole('checkbox');
```

File: assets/demo-todo-app.spec.ts [hunk 4]
```diff
@@ -167,12 +167,15 @@ test.describe('Item', () => {
 
     const firstTodo = page.getByTestId('todo-item').nth(0);
     const secondTodo = page.getByTestId('todo-item').nth(1);
-    await firstTodo.getByRole('checkbox').check();
+    const firstTodoCheckbox = firstTodo.getByRole('checkbox');
+    const secondTodoCheckbox = secondTodo.getByRole('checkbox');
```
Comment (dgozman) [inline assets/demo-todo-app.spec.ts:171]: Unused!

File: assets/demo-todo-app.spec.ts [hunk 5]
```diff
@@ -316,8 +319,10 @@ test.describe('Persistence', () => {
     }
 
     const todoItems = page.getByTestId('todo-item');
-    await todoItems.nth(0).getByRole('checkbox').check();
+    const firstTodoCheck = todoItems.nth(0).getByRole('checkbox')
```
Comment (dgozman) [inline assets/demo-todo-app.spec.ts:322]:
```suggestion
    const firstTodoCheck = todoItems.nth(0).getByRole('checkbox');
```

**Review by dgozman — APPROVED** [review]

**Pull request merged**